packages/traceability/issuer-irec-api/src/pods/certificate/certificate.controller.ts

```diff
@@ -39,4 +52,27 @@ export class IrecCertificateController extends CertificateController {
     ): Promise<IrecAccountItemDto[]> {
         return await this.commandBus.execute(new GetIrecCertificatesToImportCommand(user));
     }
+
+    @Post('/import-irec-certificate')
+    @UseGuards(AuthGuard(), ActiveUserGuard, RolesGuard, ActiveOrganizationGuard)
+    @Roles(Role.OrganizationAdmin, Role.OrganizationDeviceManager)
+    @ApiBody({ type: ImportIrecCertificateDTO })
+    @ApiResponse({
+        status: HttpStatus.CREATED,
+        type: DeviceDTO,
+        description: 'Imports a certificate from IREC'
+    })
+    @ApiForbiddenResponse({
+        status: HttpStatus.FORBIDDEN,
+        description: `User doesn't have the correct permissions`
+    })
+    @ApiBadRequestResponse({ status: HttpStatus.BAD_REQUEST })
```

Comment:
Same here + missing some description on why it could happen.

Reply:
removed